gst/nnstreamer/tensor_query/tensor_query_server.c

```diff
@@ -33,35 +33,35 @@ static void init_queryserver (void) __attribute__((constructor));
 static void fini_queryserver (void) __attribute__((destructor));
 
 /**
- * @brief Getter to get nth GstTensorQueryServerInfo.
+ * @brief Get nnstreamer edge server handle.
  */
-query_server_info_handle
-gst_tensor_query_server_get_data (guint id)
+edge_server_handle
+gst_tensor_query_server_get_handle (char *id)
 {
-  gpointer p;
+  edge_server_handle p;
 
   G_LOCK (query_server_table);
-  p = g_hash_table_lookup (_qs_table, GUINT_TO_POINTER (id));
+  p = g_hash_table_lookup (_qs_table, GUINT_TO_POINTER (*id));
```

Comment:
will you use string ID? then plz update related source about '__qs_table'.(str-hash and strdup for key and value)